### ISSUE TITLE
v0.6.0 is creating issues in ubuntu20

### ISSUE THREAD

**Issue by scarshad** [planetscale/vtprotobuf#134]
Hey
I have a jenkins CI/CD pipeline (Ubuntu 20.0) and this pipeline was working well on v0.5.0 but after upgrading to the v0.6.0 it starts breaking.

For now I am running the pipeline with v0.5.0 but I am little amazed by the error log.

```
protoc-gen-go-vtproto: program not found or is not executable
Please specify a program using absolute path or make sure the program is available in your PATH system variable
--go-vtproto_out: protoc-gen-go-vtproto: Plugin failed with status code 1
```

Also `which protoc-gen-go-vtproto` command also not outputting anything after the upgrade

**Comment by vmg** [comment]
I'm also very confused by this, and I haven't seen any other reports. Did you perhaps did something wrong with the upgrade? It looks like you haven't actually `go install`ed the plug-in, but that's probably related to the way you're setting up your CI environment, not to the release itself. Can you please double check that `go install` is working properly and that if you install other Go tools they show up in the `$PATH`?

**Comment by scarshad** [comment]
> Did you perhaps did something wrong with the upgrade?

No, I haven't touched the script from months. For installing I was using `@latest` tag and now I am using `v0.5.0` due to same error.

>It looks like you haven't actually go installed the plug-in, but that's probably related to the way you're setting up your CI environment, not to the release itself. Can you please double check that go install is working properly and that if you install other Go tools they show up in the $PATH?

I also thought same I am using Go through plugin which is accessible. But other lib are working well. So I am using proto-gen-go, grpc one too. These two are installing and working as they are working before but vtprotobuf have some issue after upgrading to `v0.6.0` Thats why I put this here may be some changes happen in the latest version.
